defender-endpoint/mde-plugin-wsl.md

```diff
@@ -34,6 +34,12 @@ Be aware of the following before you start:
 
 3. Running a custom kernel and custom kernel command line is supported in this version. However the plug-in doesn't provide any guarantees regarding the visibility within WSL when running a custom kernel and custom kernel command line.
 
+4. OS Distribution is displayed **None** in Device overview page of WSL device in Microsoft Defender portal.
+
+5. `MDE Connectivity Test: invalid` in healthcheck output indicates that the client connectivity from WSL to Defender for Endpoint service URLs is failing.
```

Comment:
It will be better to place this next to - 
If successful, you can see that the connectivity test was successful.

```If failed, you can see that the connectivity test was invalid.```